js/bitbank.js

```diff
@@ -658,11 +658,50 @@ module.exports = class bitbank extends Exchange {
             'amount': amount,
         };
         const response = await this.privatePostUserRequestWithdrawal (this.extend (request, params));
+        //
+        //     {
+        //         "success": 1,
+        //         "data": {
+        //             "uuid": "string",
+        //             "asset": "btc",
+        //             "amount": 0,
+        //             "account_uuid": "string",
+        //             "fee": 0,
+        //             "status": "DONE",
+        //             "label": "string",
+        //             "txid": "string",
+        //             "address": "string",
+        //             "requested_at": 0
+        //         }
+        //     }
+        //
         const data = this.safeValue (response, 'data', {});
-        const txid = this.safeString (data, 'txid');
+        return this.parseTransaction (data, currency);
+    }
+
+    parseTransaction (transaction, currency = undefined) {
+        const txid = this.safeString (transaction, 'txid');
+        currency = this.safeCurrency (undefined, currency);
```

Comment:
Let' add the parsed structure to parseTransaction in the comments, and we can also use the other fields to fill in the returned structure here.

Reply:
response sample added. about parsing individual fields - like i mentioned in previous comment.